src/devtools/client/themes/webconsole.css

```diff
@@ -893,19 +888,14 @@ a.learn-more-link.webconsole-learn-more-link {
   fill: currentColor;
 }
 
-.webconsole-app .message.paused.paused-before {
-  border-top: 1px solid var(--purple-50);
-  margin-top: 0px;
-}
```

Comment:
The string "paused-before" does not appear anywhere in the Replay codebase so I also cleaned up some of the unnecessary references to it.